pages/docs/getting-started/beta/quick-installation.md

```diff
@@ -10,40 +10,101 @@ Installing Open Data Hub requires OpenShift Container Platform version 4.8+. All
 
 ### Installing the Open Data Hub Operator
 
+You can either install the Open Data Hub Operator via Web GUI or via a subscription yaml. Decide that works best in your environment. Important to note is that we operate in namespace/project openshift-operators, whether installing the operator via GUI or subscription yaml.
+
+#### Installing the Open Data Hub Operator via Openshift Web GUI
 The Open Data Hub operator is available for deployment in the OpenShift OperatorHub as a Community Operators. You can install it from the OpenShift web console by following the steps below:
 
 1. From the OpenShift web console, log in as a user with `cluster-admin` privileges.  For a developer installation from [try.openshift.com](https://try.openshift.com/), the `kubeadmin` user will work.
 ![Log in to OpenShift]({{site.baseurl}}/assets/img/pages/docs/quick-installation/login.png "Log in to OpenShift")
-1. Create a new namespace named 'odh' for your installation of Open Data Hub.
+1. Create a new namespace, e.g. named 'odh' for your installation of Open Data Hub KfDefs later on.
 ![Create Namespace 'odh']({{site.baseurl}}/assets/img/pages/docs/quick-installation/create-namespace.png "Create Namespace 'odh'")
+1. Switch to the namespace 'openshift-operators' via navigation pane 'Home' - 'Projects'
 1. Find `Open Data Hub` in the `OperatorHub` catalog.
-   1. Select the new namespace if not already selected.
    1. Under `Operators`, select `OperatorHub` for a list of operators available for deployment.
    1. Filter for `Open Data Hub` or look under `AI/Machine Learning` for the icon for `Open Data Hub`.
 ![OperatorHub]({{site.baseurl}}/assets/img/pages/docs/quick-installation/operator-hub.png "OperatorHub")
 1. Click the `Install` button and follow the installation instructions to install the Open Data Hub operator.
 ![Install]({{site.baseurl}}/assets/img/pages/docs/quick-installation/install.png "Install")
-1. The subscription creation view will offer a few options including *Update Channel*, keep the `rolling` channel selected.
+1. The subscription creation view will offer a few options including *Update Channel*, make sure the `stable` channel is selected. Notice that the installation mode is 'All Namespaces in the cluster (default)'.
 ![Select Channel]({{site.baseurl}}/assets/img/pages/docs/quick-installation/channels.png "Install")
-1. It is important to switch to the namespace you created earlier. Installing in the "OpenShift Operators" namespace will result in a fatal error similar to "csv created in namespace with multiple operatorgroups, can't pick one automatically".
```

Comment:
Until we fix this "bug", this is required text to make the user aware that they ensure that the target namespace in the UI is not `openshift-operators` before creating the kfdef.